content/en/monitors/notify/variables.md

```diff
@@ -303,6 +202,153 @@ This is the escalation message @dev-team@company.com
 {{% /tab %}}
 {{< /tabs >}}
 
+
+## Notification message enrichment
+
+### Attribute and tag variables
+
+#### Multi-alert tag variables
+
+Tag variables can be used in multi-alert monitors based on the tags selected in the multi-alert group box. This works for any tag following the `key:value` syntax.
+For example, if your monitor triggers for each `env`, then the variable `{{env.name}}` is available in your notification message.
+
+For any `key:value` tag, the variable `{{key.name}}` renders `value` in the alert message. If a group is tagged with multiple `values` associated with the same `key`, the alert message renders a comma-separated string of all values, in the lexicographic order.
+
+Variable content is escaped by default. To prevent content such as JSON or code from being escaped, use triple braces instead of double braces, for example: `{{{event.text}}}`.
+
+##### Query group by host
+
+If your monitor triggers an alert for each `host`, then the tag variables `{{host.name}}` and `{{host.ip}}` are available as well as any host tag that is available on this host.
+To see a list of tag variables based on your tag selection, click **Use message template variables** in the **Say what's happening** section.
+Some specific host metadata are available as well:
+
+- Agent Version : {{host.metadata_agent_version}}
+- Machine : {{host.metadata_machine}}
+- Platform : {{host.metadata_platform}}
+- Processor : {{host.metadata_processor}}
+
+##### Query group by a facet
```

Comment:
Not sure that this one should be its own section. It's definitely part of the group by element and is a common case that should show up before the group by host which is a specific one.

Reply:
We could potentially use tabs to get tags and attribute template variable separated instead of two sections. WDYT?